src/popup/router/components/LinkButton.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <button class="link-button" @click="$emit('click')">
```

Comment:
if it is a link, why don't accept URL? proposing to name it `to` as in `RouterLink`
also, as a root node use
```suggestion
  <a class="link-button" :href="to" target="blank" @click="IS_CORDOVA && open(...)">
```
to be able to highlight in browser status bar/copy url/open in new tab and so on in the web version.